understanding/keyboard.html

```diff
@@ -55,248 +53,177 @@
-                        to learn different interaction methods. As a <em>best practice</em>, content
-                        should follow the platform/user agent conventions. However, deviating from these
-                        conventions does <em>not</em> fail the normative requirement of this Success Criterion.
+                     <p>プラットフォーム及びユーザエージェントは通常、ウェブコンテンツ又はアプリケーションをキーボードインターフェイスでどう操作するかの慣習を持っている。コンテンツがプラットフォーム／ユーザエージェントの慣習に従わない場合、利用者は異なるインタラクション方法を学習しなければならず、使いにくくなる可能性がある。<em>ベストプラクティス</em>としては、コンテンツはプラットフォーム／ユーザエージェントの慣習に従うべきである。しかし、こうした慣習から逸脱しているからといって、この達成基準の規範的要件に失敗するとはならない。
                      </p>
                      
                      <p>
-                        For instance, buttons that have focus can generally be activated using both the
-                        <kbd>Enter</kbd> key and the <kbd>Space</kbd> bar. If a custom button control
-                        in a web application instead only reacts to <kbd>Enter</kbd>
-                        (or even a completely custom key or key combination), this still
-                        <strong>satisfies</strong> the requirements of this Success Criterion.
-                        
+                        例えば、フォーカスを持つボタンは、一般的に <kbd>Enter</kbd> キーと<kbd>スペースバー</kbd>のどちらを用いても実行することができる。ただしウェブアプリケーションのカスタムボタンコントロールが <kbd>Enter</kbd> キー (あるいは、完全にカスタムなキー又はキーの組み合わせ) にしか反応しない場合でも、この達成基準の要件を<strong>満たす</strong>ことになる。
+
                      </p>
                      
                   </div>
                </div>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      
-                     <p>This Success Criterion does not require that every visible control that can be activated
-                        using a mouse or touch screen must also be focusable and actionable using the keyboard.
-                        The normative requirement is only that there must be a way for keyboard interface
-                        users to perform
-                        the same, or comparable, actions and to operate the content. Generally, the easiest
-                        way
-                        to achieve this is to provide controls that can be operated with all possible input
-                        devices;
-                        however, if a web application implements a separate mode of operation for keyboard
-                        interface users,
-                        it will <strong>not</strong> fail the Success Criterion.
+                     <p>この達成基準は、マウス又はタッチスクリーンを用いて実行できる全ての可視コントロールが、キーボードを用いてフォーカス可能かつ操作可能でなければならないことを要求するものではない。規範的な要件としては、キーボードインターフェースの利用者が、同じ又は同等のアクションを実行し、コンテンツを操作する方法がなければならないということだけである。一般的に、これを達成する最も簡単な方法は、想定されるすべての入力デバイスで操作可能なコントロールを提供することである。ただし、ウェブアプリケーションがキーボードインターフェイス利用者のために個別の操作モードを実装している場合でも、この達成基準への失敗と<strong>はならない</strong>。
```

Comment:
ウィズダムにしか見当たりませんが、このhoweverは、下記の訳語がしっくりくるのではと思いますがいかがでしょう?

> 3〚本題復帰〛（くだけて）（いろいろな意見を述べた後に本論に戻そうとして）どのようであっても, いずれにしても

```suggestion
                     <p>この達成基準は、マウス又はタッチスクリーンを用いて実行できる全ての可視コントロールが、キーボードを用いてフォーカス可能かつ操作可能でなければならないことを要求するものではない。規範的な要件としては、キーボードインターフェースの利用者が、同じ又は同等のアクションを実行し、コンテンツを操作する方法がなければならないということだけである。一般的に、これを達成する最も簡単な方法は、想定されるすべての入力デバイスで操作可能なコントロールを提供することである。いずれにせよ、ウェブアプリケーションがキーボードインターフェイス利用者のために個別の操作モードを実装している場合、この達成基準への失敗とは<strong>ならない</strong>。
```